src/events/DistanceEvents.ts

```diff
@@ -1,10 +1,17 @@
-export interface IPosition {
+/**
+ * The `IGeolocationPosition` interface describes a geolocation position.
+ */
+export interface IGeolocationPosition {
   longitude: number;
   latitude: number;
 }
 
-export interface IPositionProvider {
-  getPosition(): Promise<IPosition>;
+/**
+ * The `IGeolocationPositionProvider` interface describes a class that can provide a
```

Comment:
`class` => `handler`? Or simply `object`?

Reply:
I see that using class could be confusing since most of our users are integration the searchUI with javascript. 

Maybe we should write "object with a method that can provide..." although I'm not sure. @fbeaudoincoveo any input on this?

We could also just write "a class or an object".

Reply:
As @dms1lva says, we typically say that an interface describes "an object" rather than "a class".

I think the best way to put this would be, as Dario suggests:

> The `IGeolocationPositionProvider` interface describes an object with a method that can provide a geolocation position to the [`DistanceResources`]{@link DistanceResources} component.